build_integration_test.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+cli="./faas-cli"
+template="python3"
+if [ $(uname) == 'Darwin' ] ; then
+    cli="./faas-cli-darwin"
+elif [ -f "/etc/debian_version" ] ; then
```

Comment:
What about the approach we use in the CLI get script? I think it might be more portable across platforms?